imap/config.c

```diff
@@ -131,6 +131,9 @@ static struct ConfigDef ImapVars[] = {
   },
 
   { "imap_servernoise", DT_SYNONYM, IP "imap_server_noise", IP "2021-02-11" },
+  { "imap_send_id", DT_BOOL, false, 0, NULL,
+    "(imap) Send ID command when logging in"
+  },
```

Comment:
Move from the "synonym" section to the "non-synonym" section above (i.e. ~5 lines up).  They are both alphabetically sorted.